src/util/bip158.rs

```diff
@@ -104,14 +104,21 @@ pub struct BlockFilter {
     pub content: Vec<u8>
 }
 
+impl FilterHash {
+    /// compute the filter header from a filter hash and previous filter header
+    pub fn filter_header(&self, previous_filter_header: &FilterHeader) -> FilterHeader {
```

Comment:
I think usually we pass those hash newtypes as values because they are `Copy`. Not sure if that's a strong requirement actually.

Reply:
I guess this method in particular will be inlined by the compiler anyway.